docs/configuring/README.md

```diff
@@ -202,6 +202,29 @@ And in YAML:
 
 **Note:** A globally-installed instance of ESLint can only use globally-installed ESLint plugins. A locally-installed ESLint can make sure of both locally- and globally- installed ESLint plugins.
 
+## Configuring Preprocessors
+
```

Comment:
Hmmm, I'm not sold on needing to configure preprocessors this way. Why not just have the plugin day which files it should apply to?